docs/commands.rst

```diff
@@ -0,0 +1,67 @@
+Commands
+========
+
+A command-line tool to help manage EthPM packages and registries.
+
+
+.. warning::
+
+   ``EthPM CLI`` is currently in public Alpha:
+
+   - It is expected to have bugs and is not meant to be used in production 
+   - Things may be ridiculously slow or not work at all
+
+
+ethpm install
+-------------
+
+Install an ethPM package to a local ``_ethpm_packages`` directory.
+
+.. argparse::
+   :ref: ethpm_cli.parser.parser
+   :prog: ethpm
+   :path: install
+
+
+ethpm list
+----------
+
+List all installed ethPM packages in a local ``_ethpm_packages`` directory.
+
+.. argparse::
+   :ref: ethpm_cli.parser.parser
+   :prog: ethpm
+   :path: list
+
+
+ethpm uninstall
+---------------
+
+Uninstall an ethPM package from a local ``_ethpm_packages`` directory.
+
+.. argparse::
+   :ref: ethpm_cli.parser.parser
+   :prog: ethpm
+   :path: uninstall
+
+
+ethpm auth
+----------
+
+Link a keyfile to authorize on-chain transactions (i.e. deploying a registry / releasing a package). To generate a keyfile, use `eth-keyfile <https://github.com/ethereum/eth-keyfile>`_.
```

Comment:
Might be worth providing a full example python script to make this extra easy.

Reply:
Thoughts on just providing a cli command to do this?
```
ethpm auth generate --private-key xxx --password yyy
```